EventsExpress/Controllers/EventController.cs

```diff
@@ -121,6 +125,27 @@ public async Task<IActionResult> Edit(Guid eventId, EventEditViewModel model)
         [UserAccessTypeFilterAttribute]
         public async Task<IActionResult> Publish(Guid eventId)
         {
+            EventDto eventDto = _eventService.EventById(eventId);
+            if (eventDto == null)
+            {
+                throw new EventsExpressException("Event not found!");
+            }
+
+            EventViewModel ev = _mapper.Map<EventViewModel>(eventDto);
+
+            var validationResult = _validator.Validate(ev);
+            if (!validationResult.IsValid)
+            {
```

Comment:
this part can be moved into a separate private method... for instance `FormatValidationResult()`

Reply:
yes it would be better, already did something

Reply:
done